server/src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -309,14 +309,15 @@ static void stop() throws IOException {
     /**
      * This method is invoked by {@link Elasticsearch#main(String[])} to startup elasticsearch.
      */
```

Comment:
We should isolate this, otherwise other forbidden uses can creep into bootstrap init.